examples/src/adult-income/docker-compose.yml

```diff
@@ -8,15 +8,12 @@ services:
   data_loader:
     env_file:
       - .docker.env
-    environment:
-        TASK_SLOT_ID: "{{.Task.Slot}}"
-        REPLICAS: 1
     depends_on:
       - nn_worker
       - embedding_worker
       - persia_nats_service
-    image: ${IMAGE_PREFIX}persia-cuda-runtime:${IMAGE_TAG}
-    command: persia-launcher data-loader /workspace/data_loader.py
+    image: persiaml/persia-cuda-runtime:latest
+    command: persia-launcher data-loader /workspace/data_loader.py --replica-index 0 --replica-size 1
```

Comment:
How about add some comments to show how to launch with environment variables? 
e.g. `# export PERSIA_DATALOADER_ENTRY=/workspace/data_loader.py --replica-index 0 --replica-size 1`.

So that we can keep consistent with our tutorial.